tests/atlocal.in

```diff
@@ -81,28 +83,33 @@ fi
 
 function setup_env()
 {
-    if [ -d testing ]; then
-        RPMTEST=${PWD}/testing
-        HOME=${RPMTEST}
+    if [ -d tree ]; then
+        RPMTEST=${PWD}/tree
         RPM_CONFIGDIR=${RPMTEST}/@RPM_CONFIGDIR@
-        export RPMTEST HOME RPM_CONFIGDIR
+        export RPMTEST RPM_CONFIGDIR
```

Comment:
Not overriding $HOME is the kind of change that makes this :sheep: very nervous as long as we're running outside a container. Does overriding get in the way of something in the later stages, or is it supposed to be a cleanup kind of thing? If the latter, please leave it out for now at least.

Reply:
Oh-oh, again, you're right. On one hand, we do override a bunch of paths like `_topdir` in `run()` but you can never be sure that some program/script in the whole chain won't actually look at `$HOME`.

Us overriding it here for fakechroot is basically the poor-man's containerization that we're getting rid of, so while it's not a problem when you run the whole thing in a container, we're not there *yet* with this PR, so better leave that out, indeed.

And yup, it was a cleanup change, no other reason, really :smile: